packages/js/customer-effort-score/src/customer-feedback-simple/customer-feedback-simple.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import {
+	createElement,
+	Fragment,
+	useEffect,
+	useState,
+} from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { Button, Tooltip } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import { __ } from '@wordpress/i18n';
+
+type CustomerFeedbackSimpleProps = {
+	recordScoreCallback: ( score: number ) => void;
+	label: string;
+	feedbackScore?: number;
+	showFeedback?: boolean;
+};
+
+/**
+ * Provides a modal requesting customer feedback.
+ *
+ * A label is displayed in the modal asking the customer to score the
+ * difficulty completing a task. A group of radio buttons, styled with
+ * emoji facial expressions, are used to provide a score between 1 and 5.
+ *
+ * A low score triggers a comments field to appear.
+ *
+ * Upon completion, the score and comments is sent to a callback function.
+ *
+ * @param {Object}   props                     Component props.
+ * @param {Function} props.recordScoreCallback Function to call when the results are sent.
+ * @param {string}   props.label               Question to ask the customer.
+ * @param {number}   props.feedbackScore       Feedback score.
+ * @param {boolean}  props.showFeedback        Show feedback.
+ */
+const CustomerFeedbackSimple: React.FC< CustomerFeedbackSimpleProps > = ( {
+	recordScoreCallback,
+	label,
+	feedbackScore = NaN,
+	showFeedback,
+} ) => {
+	const options = [
+		{
+			tooltip: __( 'Very difficult', 'woocommerce' ),
+			value: 1,
+			emoji: '😔',
+		},
+		{
+			tooltip: __( 'Difficult', 'woocommerce' ),
+			value: 2,
+			emoji: '🙁',
+		},
+		{
+			tooltip: __( 'Neutral', 'woocommerce' ),
+			value: 3,
+			emoji: '😑',
+		},
+		{
+			tooltip: __( 'Good', 'woocommerce' ),
+			value: 4,
+			emoji: '🙂',
+		},
+		{
+			tooltip: __( 'Very good', 'woocommerce' ),
+			value: 5,
+			emoji: '😍',
+		},
+	];
+
+	const [ score, setScore ] = useState( feedbackScore || NaN );
+
+	useEffect( () => {
+		if ( feedbackScore !== score ) {
+			setScore( feedbackScore );
+		}
+	}, [ feedbackScore ] );
+
+	useEffect( () => {
```

Comment:
Yeah I agree, the main reason I added this is because I also added the `We appreciate your feedback` part in here, same reason for the `feedbackScore` in your comment above.
I could remove both of those if I take that out and have a parent component handle that, which in theory keeps more inline with the `Simple` approach.

Reply:
I guess this was a response to the wrong `useEffect`

Reply:
Removed this